.github/workflows/on_merge_pull_request.yml

```
@@ -38,9 +47,7 @@ jobs:
       # TODO PROJECT-SETUP Verify app distribution groups
       APP_DISTRIBUTION_GROUPS: futured-devs, futured-qa
       # TODO PROJECT-SETUP verify product flavor configuration
-      # Specifies API environment for KMP build.
-      # One of [dev|prod] as per configuration of Buildkonfig plugin in :shared:network:* Gradle module.
-      KMP_FLAVOR: dev
+      KMP_FLAVOR: ${{ env.KMP_FLAVOR }}
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)